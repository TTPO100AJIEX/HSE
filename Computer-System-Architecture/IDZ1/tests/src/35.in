100
255 102 490 -388 -426 209 438 83 -117 218 -225 -437 84 494 486 -262 -64 -480 -258 374 -213 -244 245 -189 13 89 -275 280 -212 112 -338 -493 152 -184 441 419 -352 -57 240 315 421 -285 -381 416 319 -460 388 148 193 -253 -60 38 247 1 200 -175 30 2 -433 -122 -411 397 -129 -74 -9 76 -449 175 -429 39 -378 331 -414 489 -383 267 327 -339 398 -197 -457 53 -205 -19 -82 19 -187 -173 35 368 28 66 -147 207 213 399 -64 25 249 381