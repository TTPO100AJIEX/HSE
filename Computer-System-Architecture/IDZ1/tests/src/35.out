102 490 -426 209 438 -117 218 -437 84 494 -262 -64 -480 -258 374 -244 245 -189 13 89 -275 280 -212 112 -493 152 -184 441 -352 -57 240 315 421 -381 416 -460 388 148 193 -253 -60 38 247 1 200 -175 30 -433 -122 -411 397 -129 -74 -9 76 -449 175 -429 39 -378 331 -414 489 -383 267 327 -339 398 -457 53 -205 -19 -82 19 -187 -173 35 368 28 66 -147 207 213 399 -64 25 249 381 